src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -36,6 +36,24 @@ public FileInfo GetNonZeroMilliSec()
             return fileinfo;
         }
 
+        public FileInfo GetNonZeroNanoSeconds()
+        {
+            FileInfo fileinfo = new FileInfo(GetTestFilePath());
+            for (int i = 0; i < 5; i++)
+            {
+                fileinfo.Create().Dispose();
+                if (fileinfo.LastWriteTime.Ticks % 10 != 0)
```

Comment:
Rather than writing `.Ticks % 10 != 0` in a bunch of places, how about a little helper function:
```C#
private static bool HasNonZeroNanoSeconds(DateTime dt) => dt.Ticks % 10 != 0;
```
?